new-frontiers-14/frontier-station-14
# Rosy Mothroach

## About the PR



Adds a new mob, the rosy mothroach.  It's a mothroach but with pink and yellow wings, a pink face, and pink legs.  Available from all good sources of mothroaches as a rare spawn.  Should appear in all its uses as the mothroach appears.

## Why / Balance


The rosy maple moth is pretty cute, and I thought it'd be nice to have a mothroach version of it: https://www.butterfliesandmoths.org/species/Dryocampa-rubicunda

## How to test


1. Spawn a rosy mothroach, put it on your head.
2. Spawn another, a bun, and a microwave.  Cook all together for 10 seconds.
3. Spawn a few mothroach cubes, spray them.  You should eventually get a rosy mothroach.
4. Inspect the rosy mothroach sprites.

## Media


![image](https://github.com/user-attachments/assets/a1c3a61c-bf92-403b-bf50-d3aa047144a7)


## Breaking changes


**Changelog**



:cl:
- add: The rosy mothroach, a new animal NPC, has been added in-game.
